tests/legacy-cli/e2e/tests/build/bundle-budgets.ts

```diff
@@ -17,7 +17,7 @@ export default async function () {
     ];
   });
 
-  const errorMessage = await expectToFail(() => ng('build'));
```

Comment:
`errorMessage` is an `Error` and this was relying on `errorMessage.toString()` in the `.test` below. When making the `expectToFail` return type stricter this becomes an error.